msteams-platform/toolkit/add-single-sign-on.md

```diff
@@ -114,8 +114,8 @@ The following table lists the changes Teams Toolkit makes to your project:
 
 | **Type** | **File**                                             | **Purpose**                                                                                                                                                                               |
 | -------- | ---------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| Create   | `aad.template.json` under `template\appPackage`      | Azure AD application manifest represents your Azure AD app. `template\appPackage` helps to register an Azure AD app during local debug or provision stage.                                |
-| Modify   | `manifest.template.json` under `template\appPackage` | A `webApplicationInfo` object is added into your Teams app manifest template. Teams requires this field to enable SSO. The change is in effect when you trigger local debugging or provisioning. |
+| Create   | `aad.template.json` under `template\appPackage`      | Azure AD application app manifest represents your Azure AD app. `template\appPackage` helps to register an Azure AD app during local debug or provision stage.                                |
```

Comment:
This row (Create | `aad.template.json`|...) is about AAD manifest, which is out of scope for the updates here. Could we revert the changes?

Reply:
Updated.